lib/binary-buffer.js

```diff
@@ -0,0 +1,39 @@
+"use strict";
+
+module.exports = () => {
+    /**
+     * Converts a string into a buffer. Does not change the input if it is
+     * already a buffer.
+     *
+     * @param {(Buffer|string)} stringOrBuffer
+     * @return {(Buffer|string)}
+     */
+    function toBuffer(stringOrBuffer) {
+        if (!Buffer.isBuffer(stringOrBuffer)) {
+            stringOrBuffer = new Buffer(stringOrBuffer.toString(), "binary");
```

Comment:
Eslint is barking about the first change. Should I add `/* eslint-disable consistent-return */`to the top of the file as well (`// eslint-disable-next-line consistent-return` was not working)?

`.toString()` has been removed.